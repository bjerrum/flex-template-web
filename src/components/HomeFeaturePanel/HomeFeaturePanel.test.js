import React from 'react';
import { currencyConfig } from '../../util/test-data';
import { fakeIntl } from '../../util/test-data';
import { renderShallow } from '../../util/test-helpers';
import HomeFeaturePanel from './HomeFeaturePanel';

describe('HomeFeaturePanel', () => {
  it('matches snapshot', () => {
    const props = {
      intl: fakeIntl,
    };
    const tree = renderShallow(<HomeFeaturePanel {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
