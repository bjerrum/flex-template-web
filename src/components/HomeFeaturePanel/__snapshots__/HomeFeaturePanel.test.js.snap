// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeFeaturePanel matches snapshot 1`] = `
<div
  className=""
>
  <div />
</div>
`;
